h5glance/terminal.py

```diff
@@ -249,13 +249,15 @@ def display_h5_obj(file: h5py.File, path=None, expand_attrs=False, slice_expr=No
     else:
         sys.exit("What is this? " + repr(obj))
 
-    # If the output has more lines than the terminal, display it in a pager
     output = sio.getvalue()
-    if sys.stdout.isatty():
-        nlines = len(output.splitlines())
-        _, term_lines = get_terminal_size()
-        if nlines > term_lines:
-            return page(output)
+
+    if use_pager:
+        # If the output has more lines than the terminal, display it in a pager
+        if sys.stdout.isatty():
```

Comment:
These two conditions could be combined with an `and`, which seems a bit neater. :slightly_smiling_face: